runtime/integration-tests/Cargo.toml

```diff
@@ -10,54 +10,71 @@ codec = { package = "parity-scale-codec", version = "3.0.0" }
 scale-info = { version = "2.1.2", features = ["derive"] }
 serde = { version = "1.0.144", features = ["derive"] }
 paste = "1.0.14"
+log = { version = "0.4.17" }
```

Comment:
OR, can we just remove the `trace!` code in line 332 of sibling.rs? 

I was thinking, that we really don't need any more logs in this testing package

Reply:
You're right, we don't even have the log enabled. We took this from the runtime's xcm configs as-is in the past probably.